test/types/mongodb.test-d.ts

```diff
@@ -23,10 +23,14 @@ expectDeprecated(Db.prototype.unref);
 expectDeprecated(MongoDBDriver.ObjectID);
 expectNotDeprecated(MongoDBDriver.ObjectId);
 
+// Unfortunately we cannot deprecate the export will also satisfying our tooling that requires every public symbol be exported
```

Comment:
Let's add the additional context you (@nbbeeken ) provided over slack to this comment.  If I had to ask about it, the comment probably isn't descriptive enough 🙂